runner/runners/setup.go

```diff
@@ -21,6 +22,9 @@ func (m module) Install(b *ice.MagicBag) {
 		func(m execer.Memory, s stats.StatsReceiver) execer.Execer {
 			return execers.MakeSimExecerInterceptor(execers.NewSimExecer(), osexec.NewBoundedExecer(m, s))
 		},
+		func() runner.RunnerID {
+			return runner.EmptyID
```

Comment:
Assuming we're going to override this internally... Why not use something identifying, e.g. `exec.Command("hostname")` instead of EmptyID?

Reply:
adding simple hostname + rand int